hrm-domain/hrm-service/src/profile/profile-data-access.ts

```diff
@@ -52,6 +54,7 @@ export type IdentifierWithProfiles = Identifier & { profiles: ProfileWithCounts[
 export type ProfileWithRelationships = Profile &
   ProfileCounts & {
     identifiers: Identifier[];
+    profileFlags: ProfileFlag['id'][];
```

Comment:
I just realized that in a future iteration there will be some relationship specific data in the lookup table (ex: expiresAt date for time limited blocks). Not needed for this update, but w probably want to go ahead and return an array of objects with limited properties so that we don't have to change as much about the data structures being passed around when that change comes.